workspaces/azure-devops/plugins/azure-devops/src/components/ReadmeCard/ReadmeCard.tsx

```diff
@@ -61,6 +61,12 @@ function isNotFoundError(error: any): boolean {
   return error?.response?.status === 404;
 }
 
+function isFolderPathError(error: any): boolean {
+  return (
+    error?.response?.status === 400 && error?.cause?.message?.includes('folder')
```

Comment:
I'm not super keen on this but I also don't have any alternatives. This is for sure the only way to validate this?